app/src/main/res/menu/menu_play.xml

```diff
@@ -10,4 +10,10 @@
           android:title="@string/action_remix"
           android:orderInCategory="100"
           app:showAsAction="never" />
+    <item android:id="@+id/action_edit"
+          android:title="Edit"
+          android:orderInCategory="100"
+          app:showAsAction="never"
```

Comment:
This will need to be changed from ```never``` to ```always```, to reflect a bug that @thisandagain found last night involving the ellipse menu not showing.